docs/changelog.md

```diff
@@ -2,6 +2,13 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. 
 
+## [0.7.1] - 2019-10-23
+
+### Changed
+- `--genome_server` can now be called multiple times to add additional refgenieservers
```

Comment:
let's change this to `--genome-server` to match the way we name multiple word arguments

Reply:
Agreed.

Reply:
It actually is that way, it's just incorrect in the docs.